### DISCUSSION TITLE
502 Bad Gateway after upgrade to latest docker

### DISCUSSION THREAD

**Discussion by notflip** [immich-app/immich#6493] [Q&A]
I ran `docker compose pull && docker compose up -d` inside my immich folder on my server. As soon as the pull was ready, I got a 502 Bad Gateway error. 

I'm using Nginx Proxy Manager, I checked the IP address, the port and ip are still set to `docker-ip:3001`. Did anything change? Does anyone know what could be the reason of this? 

The latest log of immich_server outputs server started at :3001, no errors. Thanks!

**Comment by alextran1502** [comment]
the port binding is actually 2283, 3001 is the internal port. So try to access your instance at port 2283

**Reply by notflip** [reply]
Immich ran fine on port :3001 for 2 months before the update, did anything change? Thanks for your reply.

**Comment by notflip** [comment]
Found ths issue, I'm using nginx proxy manager inside docker so the 3001 (internal port) is fine for me, **the issue is. that on docker compose restart a new internal IP is assigned, hence the 502. Thanks**
